### ISSUE TITLE
Extensions page not rendering [BUG]

### ISSUE THREAD

**Issue by muaawiyahtucker** [lnbits/lnbits#1538]
**Describe the bug**
Installed LNBits in line with Option 4 of [https://github.com/lnbits/lnbits/wiki/LNbits-Documentation#additional-guides](url), and installed the necessary plugins. However, when I go to the extensions page All I see is a blank screen.

https://user-images.githubusercontent.com/35174370/220685732-7370b879-3519-4979-912d-fde1433a5c89.mp4

When I look at the console.log of the page I see this:

![Screenshot 2023-02-22 at 15 34 17](https://user-images.githubusercontent.com/35174370/220685933-46ccfae0-2ad9-458e-bf2c-ef0fb3fcc240.png)

I am able to send and receive sats fine, the wallets work fine, and I'm even able to add extensions via admin UI, but when I go to use them, it just doesn't show up.

**To Reproduce**
I followed the steps in the above link and I have a video of what I did after that above

**Expected behavior**
I don't know what I should see, as it's never worked, but I guess I should see a list of extensions I've installed so I can use them.

**Screenshots**
https://user-images.githubusercontent.com/35174370/220685732-7370b879-3519-4979-912d-fde1433a5c89.mp4

![Screenshot 2023-02-22 at 15 34 17](https://user-images.githubusercontent.com/35174370/220685933-46ccfae0-2ad9-458e-bf2c-ef0fb3fcc240.png)

**Desktop (please complete the following information):**
 - LNbits version: most recent one, cloned and launched today
 - Database I'm using docker.

**Additional context**
I'm unable to access 'Manage Server' unless I turn LNBITS_ADMIN_UI=true. It doesn't matter if I add users to LNBITS_ADMIN_USERS.

**Comment by dni** [comment]
This is a bug. was this introduced yesterday? @motorina0 

> Additional context
>I'm unable to access 'Manage Server' unless I turn LNBITS_ADMIN_UI=true. It doesn't matter if I add users to >LNBITS_ADMIN_USERS.
That is the expected behaviour if you set LNBITS_ADMIN_UI false you should not be able to access manage server, even if you are an admin

**Comment by muaawiyahtucker** [comment]
Ok, understood about the LNBITS_ADMIN_UI part, 

So what should I do moving forward?
Also, in the instructions on option 4 and option 6 ([Instructions](https://github.com/lnbits/lnbits/wiki/LNbits-Documentation#additional-guides)) it still uses the 'lnbits-legend' reference. Is that intended?

--code--
docker run --detach --publish 5000:5000 --name lnbits-legend --volume ${PWD}/.env:/app/.env --volume ${PWD}/data/:/app/data **lnbits-legend**

**Comment by motorina0** [comment]
There are **two** different pages for extensions. From the screen recording, I understand that you should go to "Extensions" (not "Manage Extensions") and "enable" the extension for your user.

The difference between the two:

- **Manage Extensions**
  - for administrators. Normal users can see the page in read-only mode.
  - admins can install/uninstall extensions (for all the users in the system)
  - admins can activate/deactivate extensions (for all the users in the system)
    - if an extension is deactivated, then regular users will no longer be able to access it

- **Extensions**
  - for regular users (including admins)
  - here a user can enable/disable extensions of his own purpose
   - if an extension is disabled then only this user will not see it anylonger

**Comment by muaawiyahtucker** [comment]
Hi @motorina0 
Thanks for coming back to me with that excellent feedback.

I checked the recording again, from what I can see, I went to manage Extensions, and activated all the extensions, so they should have been visible in the Extensions tab to be able to Enable them. But the page is blank. I did click the Extensions section. What should I do now?  I'm running this on docker, on Ubuntu VPS

**Comment by motorina0** [comment]
I have added this small fix (merged in `main` already): https://github.com/lnbits/lnbits/pull/1543
Can you please let me know if it solves the issue for you.

**Comment by motorina0** [comment]
@muaawiyahtucker any luck, can we close the issue?

**Comment by muaawiyahtucker** [comment]
I re-installed from scratch, from the Option 4 method (from [Installation Instructions](https://github.com/lnbits/lnbits/wiki/LNbits-Documentation#additional-guides) and this was my results

https://user-images.githubusercontent.com/35174370/221157473-214d6d09-71df-4a9b-be71-02dd65b7ff7d.mp4

I tried it on three browsers, Safari, Impervious and Brave. Initially safari still had the same issue, but when I began to make the above screen recording, it started to work. It may have just been a cache issue why it persisted for a short period of time, but when I tried with other browsers it worked flawlessly.

So in conclusion, it seems to work perfectly fine.
